src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,98 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: string;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}: Props) => {
+  const amountOfPages = Math.ceil(total / +perPage);
+  const pageList = [];
+  const itemList = [];
+
+  // eslint-disable-next-line no-plusplus
+  for (let i = 1; i <= amountOfPages; i++) {
```

Comment:
That looks like an overcomplication here, why don't you just create pages: 
`  const pages = new Array<number>(pagesAmount).fill(0).map((_, i) => i + 1);`

and then map through them creating the same li > a structure inside your render?